### DISCUSSION TITLE
Displaying Edited Notes as a list, one per line

### DISCUSSION THREAD

**Discussion by wouterbatelaan-es** [TriliumNext/Trilium#2670] [Show and tell]
I really like the "Edited Notes" functionality on a day note,
but would like them as a list, one per line.
Was easy to do by adding this to the active theme note:

```
.edited-notes-list > .edited-note-line {
    display: list-item;
    margin-left: 10px;
    list-style: square;
}
```

This tool is great!

**Comment by TheBig-O** [comment]
This is a nice add. Thanks for putting it together. Makes finding the note you're looking for very easy.

**Comment by Joseto58** [comment]
Really useful. Thank you!

**Comment by 403705726** [comment]
这个，怎么使用啊

**Reply by Free2714** [reply]
1. 新建一个笔记，类型：css
2. 粘贴代码
```
.edited-notes-list > .edited-note-line {
    display: list-item;
    margin-left: 10px;
    list-style: square;
}
```
3. 添加属性：`#appCss`
4. Cltr+R刷新界面生效

**Comment by 403705726** [comment]
非常感谢您的帮助
&nbsp;


の&nbsp;☆→等
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年8月18日(星期天) 晚上8:00
收件人: ***@***.***&gt;; 
抄送: "の ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [zadam/trilium] Displaying Edited Notes as a list, one per line (Discussion #2670)














  
新建一个笔记，类型：css
 
粘贴代码
  .edited-notes-list &gt; .edited-note-line {     display: list-item;     margin-left: 10px;     list-style: square; }   
添加属性：#appCss
 
Cltr+R刷新界面生效
